src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/IRazorCodeActionResolveHandler.cs

```diff
@@ -8,7 +8,7 @@
 namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions
 {
     [Serial, Method(LanguageServerConstants.RazorCodeActionResolveEndpoint)]
```

Comment:
@NTaylorMullen do you know why this was created as `Serial` considering the CodeActionHandler from O# is `Parallel`.
```suggestion
    [Parallel, Method(LanguageServerConstants.RazorCodeActionResolveEndpoint)]
```

Reply:
> @NTaylorMullen do you know why this was created as `Serial` considering the CodeActionHandler from O# is `Parallel`.

o interesting. I don't. PRobably just an oversight. This will actually be a huge win making it parallel 👍